content/performance/docs/binpack.md

```diff
@@ -45,332 +45,25 @@ The following customer scheduler named “my-scheduler” is created for EKS ver
 
 We do not recommend build the kube-scheduler by yourself, you can leverage the eks-distro kube-scheduler image. For example:
 
-- **Amazon EKS 1.28 image:** public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v1.28.11-eks-1-28-latest
-- **Amazon EKS 1.29 image:** public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v1.29.6-eks-1-29-18
```

Comment:
Can we keep these sections as is, and then add a note/section that point to the helm chart. 
The idea is to keep the current level of details to explain what is being done in terms of RBAC, and provide the helm chart as utility to simplify the deployment.

Reply:
Done